uint32 size
geometry_msgs/PoseArray pose_array
string[] color_names

# TODO add more stuff
